ui/core/src/actions/clp.ts

```diff
@@ -5,14 +5,11 @@ import { PoolStore } from "../store/pools";
 export default ({
   api,
   store,
-}: ActionContext<
-  "SifService" | "MarketService" | "ClpService",
-  "pools" | "wallet"
->) => {
+}: ActionContext<"SifService" | "ClpService", "pools" | "wallet">) => {
   const state = api.SifService.getState();
 
   // Sync MarketService with pool store
-  api.MarketService.onPoolsUpdated((pools) => {
+  api.ClpService.onPoolsUpdated((pools) => {
```

Comment:
I'd do either `PoolService`/`poolFinder` or `ClpService`/`ClpFinder`. For more consistency in naming

Reply:
Yep great idea. It should be easier to get this in - get the other one in and then do a rename.